src/main/scala/io/pubcrawler/stout/util/JWTUtil.scala

```diff
@@ -0,0 +1,27 @@
+package io.pubcrawler.stout.util
+
+import java.time.LocalDateTime
+
+import org.json4s.JsonAST.JObject
+import io.pubcrawler.stout.db.User
+import pdi.jwt.{JwtJson4s, JwtAlgorithm}
+import org.json4s.JsonDSL.WithBigDecimal._
+
+trait JWTUtil {
+
+  val JwtSecret = "secret"
+  val HashAlgorithm = JwtAlgorithm.HS256
+
+  def createJWTToken(user: User) : String = {
+    val header = JObject(("typ", "JWT"), ("alg", "HS256"))
```

Comment:
Kun en antagelse her. Hvis du lager deg en case class med for eksempel:
case class JWTToken(val header: Header, val claim: Claim)
case class Header(val typ: String, val alg: String)
case class Claim(val iss: String, val sub: Int, val iat: String, val exp: String)

Json4s case class til json. Jeg bare antar JwtJson4s det samme.
Det hadde kanskje blitt litt ryddigere her.